src/sizeMapping.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @module sizeMapping
+ */
+import * as utils from './utils';
+
+exports.mapSizes = function(adUnit) {
+  if(!isSizeMappingValid(adUnit.sizeMapping)){
+    return adUnit.sizes;
+  }
+  const width = this.getScreenWidth();
```

Comment:
On a test page, this line gave me `TypeError: Cannot read property 'getScreenWidth' of undefined` due to this module using CommonJS-style exports but adapter manager using ES2015-style imports. Line 4 of adaptermanager could be changed to use a require statement, or the functions in this module could be exported with the ES2015 `export` keyword--in that case the unit tests fail though and would need to be changed as well